flask_monitoringdashboard/core/reporting/mean_permutation_test.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+
+def mean_diff(x, y):
+    return np.abs(np.mean(x) - np.mean(y))
+
+
+def mean_permutation_test(x, y, num_rounds=1000):
+    """
+    Performs a non-parametric test to check whether `x` and `y` come from the same distribution.
```

Comment:
Is this the [Kolmogorov-Smirnov test](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test) or another one? It would be nice if you added a link to the test you used, so the code will be easier to understand.

Reply:
Hm, I'm not sure if it has a name. It's not the one you mentioned. I just know it as "random permutation test". I'll have a look if I can a name.

Reply:
Did some googling but it seems to just be called "randomization test" or "re-randomization test"